pkg/odo/cli/catalog/describe/operator_backend.go

```diff
@@ -84,11 +85,26 @@ func (ohb *operatorBackend) RunDescribeService(dso *DescribeServiceOptions) erro
 		machineoutput.OutputSuccess(svc.ConvertCRDToJSONRepr(ohb.CR))
 		return nil
 	}
-	output, err := yaml.Marshal(svc.ConvertCRDToRepr(ohb.CR))
-	if err != nil {
-		return err
-	}
+	repr := svc.ConvertCRDToRepr(ohb.CR)
 
-	fmt.Print(string(output))
+	fmt.Printf("Kind: %s\n", repr.Kind)
```

Comment:
can we extract this into a function in this same package?

Reply:
this I meant - https://github.com/openshift/odo/pull/4812/files#diff-0d330dc955dfbc327b5a13bd4933557f7529940e0d7f756d9a186f9349a9d0b6R90-R108

Reply:
you mean in this same package `describe`? Why?

Reply:
keeping it clean?

Reply:
How it would be cleaner? It would be the same code, only in a different place.

Reply:
ok, np.